### ISSUE TITLE
Increased glutamate in urine

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6036]
**Preferred term label:** Increased glutamate in urine

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of glutamate in the urine is above the normal range.

**Parent term (use hpo.jax.org/app)** Aminoaciduria HP:0003355

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding as HP:0033094
